mlflow/genai/scorers/base.py

```diff
@@ -17,6 +17,10 @@
 
 # Serialization version for tracking changes to the serialization format
 _SERIALIZATION_VERSION = 1
+_AGGREGATION_FUNC_TYPE = Callable[[list[int | float]], float]
+_AGGREGATION_TYPES = (
```

Comment:
```suggestion
_AGGREGATION_TYPES: TypeAlias = (
```

let's use `TypeAlias` here.